### ISSUE TITLE
iPad is not rendering input textbox for lex web ui chatbot

### ISSUE THREAD

**Issue by puradravi007** [aws-samples/aws-lex-web-ui#320]
I have created a lex-web-ui and deployed as per the Readme document. Application is working in desktop without any issues. When we tried to access the lex-web-ui chatbot in iPad browsers [Safari, Chrome], we are facing issue specially on input textbox is not available.  Attached the screenshot for reference. 
Even, I have added media query to CSS for full page[index.html], still it is not working. Is there anyway we can resolve this issue so application will be accessible via all devices and across browsers. 

![Lexwebui_Keyboard](https://user-images.githubusercontent.com/64885321/132509949-a90c8eb8-4311-462c-9559-b69d10f70c55.jpg)
![Lexweb_ui_withoutKyeboard](https://user-images.githubusercontent.com/64885321/132509961-11644577-febe-424a-9be0-7ee9c4157868.jpg)

**Comment by tkashi** [comment]
I am facing similar issue that the input box disappears (and parent page is showed) in safari/chorme on iPhone Xs (iOS 14.7.1) as attached here.
![Upload from ios 2021-08-23 16_38_20 (1)](https://user-images.githubusercontent.com/1989039/133874068-a879710f-ecec-47b0-b7d1-874be4b27c80.png)

**Comment by bobpskier** [comment]
I looked at reproducing the problem using lexwebui 0.18.2 and iOS 14.8 on both an iPhone XS and iPad Pro 9.7 on 14.6.  iOS will scroll the parent page based on user swipes underneath the iframe. When the keyboard was visible, it was possible to scroll the text input area below the iOS keyboard. If I then typed text as input it would automatically scroll back to the input area.  I could also manually scroll back to see the input area if it was hidden. Is there a different behavior that is desired? iOS will naturally hide input areas on a browser as the user interacts with the keyboard and moves in and out of full page mode where the tabs and address bar are visible or hidden. I was always able to navigate back to the input area.  Have you deployed updates to the custom css file for LexWebUi or adjusted css on the parent page to place the lexwebui at a location different than the defaults?
